### PR TITLE
AK-23345 Make field names in connector_ipsec to be consitent with portal

### PR DESCRIPTION
**Pull request by spikewang**
In connector_ipsec, make field name in segment routing to be consistent with portal.

### CODE REVIEW

**Comment by spikewang** [comment]
Just update those two fields to be consistent with portal to make it less confusing.

**Review by mac-alkira — APPROVED** [review]
My comments on uniformity in description line breaks shouldn't stop you from merging this, but we should consider this at the very least for future maintenance.

File: alkira/resource_alkira_connector_ipsec.go [hunk 1]
```diff
@@ -11,11 +11,15 @@ import (
 
 func resourceAlkiraConnectorIPSec() *schema.Resource {
 	return &schema.Resource{
-		Description: "Manage IPSec Connector.",
-		Create:      resourceConnectorIPSecCreate,
-		Read:        resourceConnectorIPSecRead,
-		Update:      resourceConnectorIPSecUpdate,
-		Delete:      resourceConnectorIPSecDelete,
+		Description: "Manage IPSec Connector.\n\n\n\n" +
```
[the remark this answers is not in the document]
Comment (spikewang) [reply alkira/resource_alkira_connector_ipsec.go:14]: This is basically inline Markdown for generating documentation. That's what the tfplugindoc already does. I don't think we should have another helper doing similar thing. Hopefully, HashiCorp will have better way to generate doc down the road. I will check if anything new is available right now...

[the remark this answers is not in the document]
Comment (spikewang) [reply alkira/resource_alkira_connector_ipsec.go:14]: The current tool is just not flexible enough to cover all documentation cases we needed. Will keep checking.

**Pull request merged**